README.md

```diff
@@ -255,6 +255,12 @@ Hikari does not include a command framework by default, so you will want to pick
 - [`tanjun`](https://github.com/FasterSpeeding/Tanjun) - a flexible command framework designed to extend Hikari.
 - [`crescent`](https://github.com/magpie-dev/hikari-crescent) - a command handler for Hikari that keeps your project neat and tidy.
 
+There are also third party libraries to help you manage components:
+
+- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhances hikari.
```

Comment:
```suggestion
- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhance hikari.
```

Reply:
fyi @FasterSpeeding this was copied directly from your project description on pypi so you may have to fix that there